test/tools/mock.js

```diff
@@ -70,6 +70,6 @@ function createServer(port, host, options) {
 module.exports = {
   createServer,
   cleanup,
-  DEFAULT_ISMASTER,
-  DEFAULT_ISMASTER_36
+  LEGACY_HELLO: DEFAULT_ISMASTER,
```

Comment:
Do we actually need the LEGACY_HELLO at all if we no longer support pre-3.6? I saw most tests got updated to use `HELLO` regardless of whether they were previously `DEFAULT_ISMASTER` or `DEFAULT_ISMASTER_36`. There are only 3 files (if I counted right) that still use `LEGACY_HELLO`: find_cursor, the skipped buffering_proxy test, and the helper MongosFixture (which isn't even used in the code base). Is there any harm in eliminating these references entirely and replacing with `HELLO`?

Reply:
I've completely removed the use of `LEGACY_HELLO` now in the tests as well as the unused `MongosFixture`.